src/test/game/BoardTest.java

```diff
@@ -2,148 +2,147 @@
 
 import main.game.Board;
 import main.game.Line;
+import main.game.Mark;
 import org.junit.Test;
 
 import java.util.List;
 
 import static java.util.Arrays.asList;
+import static main.game.Mark.*;
 import static org.junit.Assert.*;
 
 public class BoardTest {
 
-    private final String PLAYER_ONE = "X";
-    private final List<String> emptyBoard = asList("1", "2", "3", "4", "5", "6", "7", "8", "9");
-
     @Test
     public void canMakeMoveOnABoard() {
-        Board board = new Board(emptyBoard);
-        List grid = board.updateMove("1", PLAYER_ONE);
+        Board board = new Board();
+        List<Mark> grid = board.updateMove(1, X);
 
-        assertEquals(PLAYER_ONE, grid.get(0));
+        assertEquals(X, grid.get(0));
     }
 
     @Test
     public void canReturnRows() {
-        Board board = new Board(emptyBoard);
+        Board board = new Board();
         List<Line> rows = board.rowLines();
-        assertEquals(createNewLine("1", "2", "3"), rows.get(0));
-        assertEquals(createNewLine("4", "5", "6"), rows.get(1));
-        assertEquals(createNewLine("7", "8", "9"), rows.get(2));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(0));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(1));
+        assertEquals(createNewLine(EMPTY, EMPTY, EMPTY), rows.get(2));
```

Comment:
I don't think this test is testing the correct thing but more importantly I'm not sure it adds much value. Your `rows` method is tested indirectly by other methods. Also is the `rows` method used anywhere else apart from the `Board` class?